common-arcana.lic

```diff
@@ -560,21 +560,20 @@ module DRCA
       data['cast'] = "cast #{cast_on}"
       return data
     end
-    echo 'Could not find any planets to cast on'
+    echo "No planet available to cast #{data['abbrev']}"
     nil
   end
 
   def set_moon_data(data)
```

Comment:
This probably isn't something we can change here, but the Ruby convention would be to name the function `set_moon_data!`

Reply:
Good to know, thanks